src/components/navigation/tests/navItems_test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import NavItems from '../navItems'
+
+describe('header', () => {
+  it('matches snapshot', () => {
+    const mockProps = {
+      items: {
+        listItems: ['Shante, you stay', 'Sashay away'],
+      },
+    }
+    const wrappedComponent = shallow(<NavItems {...mockProps} />)
+    expect(wrappedComponent).toMatchSnapshot()
```

Comment:
You can make these snapshots look nicer by using react-test-renderer instead of enzyme.

e.g.

```js
import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { shallow } from 'enzyme'

import BannerTitle from './'

describe('BannerTitle', () => {
  it('should render', () => {
    const tree = renderer.create(<BannerTitle />)
    expect(tree).toMatchSnapshot()
  })

  it('should render a <span />', () => {
    const wrapper = shallow(<BannerTitle />)
    expect(wrapper.find('span')).toHaveLength(1)
  })

  it('should render child text ', () => {
    const text = 'Here is a test title!'
    const wrapper = shallow(<BannerTitle>{text}</BannerTitle>)
    expect(wrapper.find('span').text()).toBe(text)
  })
})

```

Reply:
This seems to be a convention thing IMO, not really an issue.

Reply:
Hi @kiraarghy I think I didn't explain what I meant by 'look nicer'.

e.g. this is what the enzyme snapshots in this PR look like:

```js
...
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <Row
        alignItems="center"
        justifyContent="center"
      >
        <TestNav
          items={Object {}}
        />
      </Row>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "alignItems": "center",
        "children": <TestNav
          items={Object {}}
        />,
        "justifyContent": "center",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
...
```

While this is what snapshots with react-test-renderer look like:

```js
.c0 span {
  background-color: #ffffff;
  display: inline-block;
  padding: 0 10px;
}

<h1
  className="c0"
>
  <span />
</h1>
```

So enzyme vs react-test-render is essentially JS objects which largely represent implementation details vs html + css which represent what the user actually sees. I think the latter is both easier to read and contains information which is more useful to capture in snapshots.

Reply:
Hi David - I disagree with you on this point, but feel it is bike shedding to go back and forth on it. It certainly is community convention to use the inbuilt Enzyme tooling and as we are angling for accessible tech, let's continue with Kara's approach.

Reply:
Hi Marcel – My bad, I didn't realise there's a jest plugin for improving enzyme snapshots: https://www.npmjs.com/package/enzyme-to-json – I should have suggested we use this instead of react-test-renderer

react-test-renderer is what is used in the Jest docs examples, which is why I suggested that initially. Apologies for any confusion.